src/python/pants/help/help_integration_test.py

```diff
@@ -35,6 +35,16 @@ def test_help_targets() -> None:
     assert "to get help for a specific target" in pants_run.stdout
 
 
+def test_help_subsystems() -> None:
+    pants_run = run_pants(["--backend-packages=pants.backend.python", "help", "subsystems"])
+    pants_run.assert_success()
+    assert (
+        "pex                     How Pants uses Pex to run Python subprocesses" in pants_run.stdout
+    )
+    assert "to get help for a specific subsystem" in pants_run.stdout
+    assert "test            Run tests." not in pants_run.stdout
```

Comment:
Negative assertions are generally brittle. If we rewrite the docstring for `TestSubsystem` to be different than "Run tests.", then this assertion will always pass, even if we break things. https://thepugautomatic.com/2013/04/negative-assertions/ talks a bit about this.

I think the best thing here would be to still use a negative assertion, but to use the `re` module to make it less brittle. Maybe something like `^test\s+`. It's plausible we'll change the docstring and/or number of spaces, but very unlikely we'd ever change the line starting with `test` imo.

Reply:
Yeah, I was wondering that myself.. it wouldn't fail but putting only the word "test" is also not a good idea. I think what you said is definitely a good solution, using some regex :heart:

Reply:
> but putting only the word "test" is also not a good idea

Indeed, and this assertion wouldn't work either. `test` is part of `pytest` hehe.

Reply:
Done, updated it to use regex.